test/northstar-client.test.js

```diff
@@ -3,60 +3,89 @@
 /**
  * NorthstarClient.test
  */
+require('dotenv').config({ silent: true });
 const NorthstarClient = require('../lib/northstar-client');
+const NorthstarUser = require('../lib/northstar-user');
+const NorthstarUserAuthorized = require('../lib/northstar-user-authorized');
 
 /**
  * Test Northstar Nodejs client.
  */
 describe('NorthstarClient', () => {
-  // Constructor.
-  describe('constructor', () => {
-    // Test new instance.
-    it('should create new instance', () => {
-      const client = new NorthstarClient();
-      client.should.be.an.instanceof(NorthstarClient);
+  /**
+   * Helper: get default client.
+   */
+  function getUnauthorizedClient() {
+    return new NorthstarClient({
+      baseURI: process.env.DS_REST_API_BASEURI,
     });
-  });
-
-  describe('method', () => {
-    const client = new NorthstarClient();
-
-    // Check methods
-    it('getUser() should be exposed', () => {
-      client.getUser.should.be.a.Function();
+  }
+  /**
+   * Helper: get default client.
+   */
+  function getAuthorizedClient() {
+    return new NorthstarClient({
+      baseURI: process.env.DS_REST_API_BASEURI,
+      apiKey: process.env.DS_REST_API_KEY,
     });
-  });
-
+  }
 
   // Without X-DS-REST-API-Key.
   describe('unauthorized', () => {
     /**
      * Helper: validate unauthorized user object.
      */
     function unauthorizedTestUser(user) {
-      // TODO: check to be an instance of NorthstarUser.
-      user.should.be.an.Object();
+      // Ensure result to be an instance of NorthstarUser.
+      user.should.be.an.instanceof(NorthstarUser);
 
       // Ensure properties and test values.
       user.should.have.properties({
+        isAuthorized: false,
         id: '5480c950bffebc651c8b456f',
-        first_name: 'test',
-        last_initial: 'L',
+        firstName: 'test',
+        lastInitial: 'L',
         photo: 'https://avatar.dosomething.org/uploads/avatars/5480c950bffebc651c8b456f.jpeg',
         language: 'en-global',
         country: 'US',
-        drupal_id: '187',
+        drupalID: '187',
       });
 
       // Just ensure presence.
-      user.should.have.properties(['updated_at', 'created_at']);
+      user.should.have.properties(['updatedAt', 'createdAt']);
     }
 
+    // Constructor.
+    describe('constructor', () => {
+      // Test new instance.
+      it('without options should throw a TypeError', () => {
+        (() => new NorthstarClient()).should.throw(TypeError);
```

Comment:
No. Functions `getUnauthorizedClient()` and `getAuthorizedClient()` are helpers to construct  `NorthstarClient()` with options (`baseURI` and `apiKey`). In this case I'm building `NorthstarClient()` incorrectly, without options, to ensure throws `TypeError`.